src/Screens/SaveGame.cs

```diff
@@ -204,7 +207,10 @@ public override bool KeyDown(KeyboardEventArgs args)
 				if (_gameId >= 0)
 				{
 					SaveGameFile file = GetSaveGames().ToArray()[_gameId];
-					Game.Save(file.SveFile, file.MapFile);
+
+                    var saver = new OriginalGameSaveToFileWriter(file.SveFile, file.MapFile);
```

Comment:
SWY1985 - 'var' is a strong type in C# - the difference is that it implicitly vs explicitly typed.  It will compile to the same code if you use 'var' or the explicit name ('OriginalGameSavetoFileWriter) in this case.  https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/var

...if it's a style choice though - fair enough, since it's your codebase and your style guide.